README.md

```diff
@@ -20,7 +19,21 @@
   - [Documentation](#documentation)
 
 # OpenShift Hive
-API driven OpenShift cluster provisioning and management
+API driven OpenShift 4 cluster provisioning and management.
+
+Hive is an operator which runs as a service on top of OpenShift.
+The Hive service can be used to provision and manage OpenShift clusters.
+There is no limit on how many cluster Hive can provision or manage.
+
+* For provisioning OpenShift, Hive uses [installer](https://github.com/openshift/installer).
+* For management of hive-managed clusters, [SyncSet](./docs/syncset.md) object is used.
```

Comment:
"For day 2 configuration management, Hive provides the SyncSet CRD to deliver arbitrary Kubernetes configuration to the cluster."